_component_design/choice-tiles.md

```diff
@@ -269,16 +294,16 @@ If there is an error on a selection, change that item to errored state and displ
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Group-select error
+### Single-select error
```

Comment:
there are a few tweaks needed to the visual here too. e.g. 
- icon and option label aren't totally centered in 
the tile. 
- some say "Option Label" some say "Label"  let's standardize on "Option Label"

Reply:
roman updated the visuals, done.